samples/server/petstore/kotlin-springboot/src/main/kotlin/org/openapitools/api/PetApiServiceImpl.kt

```diff
@@ -1,17 +1,18 @@
 package org.openapitools.api
 
 import org.openapitools.model.ModelApiResponse
+import org.openapitools.model.OrgspringframeworkcoreioResource
```

Comment:
Everything looks good, but this import.

Reply:
outch, I Thought i resolved this problem (in my own company project who use open-api), i will look into it tomorrow morning, it's my first time looking into the openapi-generator code

Reply:
ok found the problème, I did not build the CLI before running the génération script (in my project I used the gradle plugin)